tools/appveyor.psm1

```diff
@@ -342,6 +348,32 @@ function Invoke-AppVeyorTest
     Set-BuildVariable -Name TestPassed -Value True
 }
 
+function Compress-CoverageArtifacts
+{
+    param([string] $CodeCoverageOutput)
+
+    # Create archive for test content, OpenCover module and CodeCoverage build
+
+    $artifacts = @()
+
+    $zipTestContentPath = Join-Path $pwd 'tests.zip'
+    Compress-TestContent -Destination $zipTestContentPath
+    $artifacts.Add($zipTestContentPath)
```

Comment:
This will cause an exception because you are adding to an array:
`Exception calling "Add" with "1" argument(s): "Collection was of a fixed size."`